packages/util-actor/src/index.ts

```diff
@@ -1,5 +1,5 @@
-import {mailbox as createMailbox} from "./mailbox"
-import queueMicrotask from "queue-microtask"
+import {mailbox as createMailbox, type IMailbox} from "./mailbox"
+const queueMicrotask = require("queue-microtask")
```

Comment:
was there a reason that this can't be an `import` statement, or is this just a mistake.  probably best that we don't mix import types throughout the project if avoidable.

Reply:
this package is not an esm, it uses `module.exports` instead of `export` statement

Reply:
i think it should be compatible to import cjs module with esm import syntax though, i.e.

`import * as queueMicrotask from "queue-microtask"`